### DISCUSSION TITLE
it/analisi-pa-2023/

### DISCUSSION THREAD

**Discussion by giscus** [matteocontrini/blog#3] [Comments]
# it/analisi-pa-2023/

Seconda edizione dell’analisi dei siti web della PA. Quest’anno ho analizzato 46 siti web ed estratto 13 parametri di valutazione.

https://blog.contrini.it/it/analisi-pa-2023/

**Comment by notherealmarco** [comment]
incredibile come la maggior parte dei problemi siano legati a record DNS mancanti o settati male 😢

Ma poi se il governo ha un suo AS, perché alcuni ministeri lo usano e altri no? 🧐

**Reply by matteocontrini** [reply]
> Ma poi se il governo ha un suo AS, perché alcuni ministeri lo usano e altri no? 🧐

Se intendi il fatto che ho scritto "AS Governo" per `www.sport.governo.it`, in realtà intendevo "AS Presidenza del Consiglio dei Ministri" ma non ci stava (ora sistemo in qualche modo). Il sito "sport" si riferisce al dipartimento per lo sport che fa parte della Presidenza (cioè `governo.it`), quindi usano quell'AS.

Per il resto, sembra che alcuni ministeri abbiano un proprio AS (Difesa, Esteri, Camera) e lo usino, gli altri non si sa. L'unica cosa certa è che come per tutto il resto è tutto rigorosamente a caso :)

**Reply by notherealmarco** [reply]
Ahh ok, credevo esistesse proprio un AS "governo" per tutti i ministeri
